projects/plugins/social/src/js/components/connection-screen/index.js

```diff
@@ -47,6 +29,37 @@ const ConnectionScreen = () => {
 		...connectProps,
 	} );
 
+	const buttonText = __( 'Get Started', 'jetpack-social' );
+	const tos = createInterpolateElement(
```

Comment:
Should we make ToS a reusable component so in the future we only need to change this component instead of doing it in multiple files?

Reply:
I second this.
Extracting it into a component would be a good practice.